lifechanger/src/app/app.module.ts

```diff
@@ -6,13 +6,23 @@ import { AppComponent } from './app.component';
 import { FooterComponent } from './footer/footer.component';
 import { HeaderComponent } from './header/header.component';
 import { HeroComponent } from './hero/hero.component';
+<<<<<<< HEAD
```

Comment:
Here is conflict, you need to fix it,also it on 21 line

Reply:
We will continue with fixing merge confilcs in our next discussion

Reply:
Thank you, dear Mentor, for resolving the conflicts:)
I repaired the below mentioned comments